infra/gcb/build_project.py

```diff
@@ -104,21 +117,29 @@ def workdir_from_dockerfile(dockerfile):
   return None
 
 
-def get_build_steps(project_dir):
-  project_yaml = load_project_yaml(project_dir)
-  dockerfile_path = os.path.join(project_dir, 'Dockerfile')
+def load_project_yaml(project_name, project_yaml_file, image_project):
+  """Loads project yaml and sets default values."""
+  project_yaml = yaml.safe_load(project_yaml_file)
+  set_yaml_defaults(project_name, project_yaml, image_project)
```

Comment:
this fails (at least with python2), but the following change makes it work:

```diff
$ git diff
diff --git a/infra/gcb/build_project.py b/infra/gcb/build_project.py
index 7798bcf2..7006cbb2 100644
--- a/infra/gcb/build_project.py
+++ b/infra/gcb/build_project.py
@@ -119,9 +119,10 @@ def workdir_from_dockerfile(dockerfile_lines):
 
 def load_project_yaml(project_name, project_yaml_file, image_project):
   """Loads project yaml and sets default values."""
-  project_yaml = yaml.safe_load(project_yaml_file)
-  set_yaml_defaults(project_name, project_yaml, image_project)
-  return project_yaml
+  with open(project_yaml_file) as f:
+    project_yaml = yaml.safe_load(f)
+    set_yaml_defaults(project_name, project_yaml, image_project)
+    return project_yaml
 
 

```

Reply:
Thanks i made the changes!